clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -52,20 +53,22 @@ jest.mock("vscode", () => ({
     }),
   },
   workspace: {},
+  window: {
+    showErrorMessage: jest.fn().mockReturnValue(""),
+  },
 }));
 
-describe(".gitignore file in .c4z folder tests", () => {
+describe(".gitignore file in extension path folder tests", () => {
```

Comment:
is this still relevant?

Reply:
We need a feedback from maintainers here. I have an hesitation that preprocessors /dialects configurations is still being kept under workspace and do we need ignore file still under this ?

Reply:
removed at : 533bc9f181e4bd6b8c63959229b92a9797bcbb99